github/README.md

```diff
@@ -16,6 +16,20 @@ Usage (can run on your Mac or within dev VM):
 ```
 GITHUB_TOKEN=xyz bundle exec rake github:configure_repos
 ```
+
+### `rake github:verify_repo_tags`
+
+Will compare the list of currently GOVUK tagged repos on GitHub with the list of repos in devdocs (https://github.com/alphagov/govuk-developer-docs/blob/main/data/repos.yml), then prints a list of untagged and
+falesly tagged repos.
+
+Requires a GitHub personal access token, which can be generated using the [GitHub UI](https://github.com/settings/tokens).
```

Comment:
PS I had to [generate a new 'classic' token](https://github.com/settings/tokens/new) with full `repo` scope permissions. I tried a couple of different 'fine-grained' tokens instead but couldn't get them to work.

Without this scope, I was seeing lots of repos in the list of "Untagged govuk repos:", which were just private repos (but _were_ tagged, e.g. `govuk-secrets`).

May be worth tweaking the README 👍